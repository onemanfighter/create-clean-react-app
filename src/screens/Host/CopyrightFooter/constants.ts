export const COPYRIGHT_FOOTER_VERTICAL_PADDING = 2;
