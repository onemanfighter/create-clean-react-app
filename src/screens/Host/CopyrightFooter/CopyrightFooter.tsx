import { Box, Text } from '@chakra-ui/react';
import { useTranslation } from 'react-i18next';
import { COPYRIGHT_FOOTER_VERTICAL_PADDING } from './constants';

const CopyrightFooter = () => {
  const { t } = useTranslation();
  const currentYear = new Date().getFullYear();
  return (
    <Box py={COPYRIGHT_FOOTER_VERTICAL_PADDING}>
      <Text textAlign={'center'}>
        {t('Footer.copyrightText', { year: currentYear })}
      </Text>
    </Box>
  );
};

export default CopyrightFooter;
