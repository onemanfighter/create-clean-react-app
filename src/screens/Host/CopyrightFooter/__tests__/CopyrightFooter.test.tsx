import { renderWithRouter } from '@testUtils';
import CopyrightFooter from '../CopyrightFooter';

describe('CopyrightFooter', () => {
  it('should render', () => {
    const { container } = renderWithRouter(<CopyrightFooter />);

    expect(container).toMatchSnapshot();
  });
});
