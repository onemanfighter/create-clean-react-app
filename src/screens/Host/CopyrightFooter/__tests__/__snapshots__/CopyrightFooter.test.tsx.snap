// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CopyrightFooter should render 1`] = `
<div>
  <div
    class="css-o5g2qp"
  >
    <p
      class="chakra-text css-xi606m"
    >
      Â© 2024 Hourcoding.com | All rights reserved
    </p>
  </div>
</div>
`;
